web_advanced_search/static/src/js/web_advanced_search.js

```diff
@@ -153,14 +156,23 @@ odoo.define("web_advanced_search", function (require) {
             // To make widgets work, we need a model and an empty record
             FieldManagerMixin.init.call(this);
             this.trigger_up("get_dataset");
-            // Make equal and not equal appear 1st and 2nd
+            // Unfortunately, it's impossible to check if model supports these
```

Comment:
Why ?  Checking if model has parent_id field is not possible ?

Reply:
Technically, we should check if `_parent_store = True` or check for field with name stored in `_parent_name` (by default, that's `parent_id`), yet I haven't found any clues on how to get those in frontend or example of those values being used in js.

Reply:
OK ! Could you just add it in the roadmap section ?

Reply:
Done